identity/src/main/java/com/stripe/android/identity/navigation/ScanDestinations.kt

```diff
@@ -1,87 +1,159 @@
 package com.stripe.android.identity.navigation
 
-import androidx.core.os.bundleOf
+import androidx.annotation.StringRes
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.NavType
+import androidx.navigation.navArgument
 import com.stripe.android.identity.R
+import com.stripe.android.identity.networking.models.CollectedDataParam
+import com.stripe.android.identity.states.IdentityScanState
 
-internal class PassportScanDestination(
+internal abstract class DocumentScanDestination(
```

Comment:
DocumentScanDestination is the super class of `PassportScanDestination`, `IDScanDestination` and `DriverLicenseScanDestination`, as their parameters list are exactly the same